wrapper/src/fmriprep_docker/__main__.py

```diff
@@ -521,7 +521,7 @@ def main():
         unknown_args.append('--derivatives')
         for deriv, deriv_path in opts.derivatives.items():
             command.extend(['-v', '{}:/deriv/{}:ro'.format(deriv_path, deriv)])
-            unknown_args.append('/deriv/{}'.format(deriv))
+            unknown_args.append('{}=/deriv/{}'.format(deriv, deriv))
```

Comment:
This is now redundant:

```suggestion
            unknown_args.append('/deriv/{}'.format(deriv))
```

Reply:
Good point!